googlepay/src/main/java/com/adyen/checkout/googlepay/util/GooglePayUtils.java

```diff
@@ -235,7 +237,9 @@ private static TransactionInfoModel createTransactionInfo(@NonNull GooglePayPara
         final String displayAmount = GOOGLE_PAY_DECIMAL_FORMAT.format(bigDecimal);
 
         final TransactionInfoModel transactionInfoModel = new TransactionInfoModel();
-        transactionInfoModel.setTotalPrice(displayAmount);
+        if (!params.getTotalPriceStatus().equals(NOT_CURRENTLY_KNOWN)) {
```

Comment:
Thanks a lot for your PR! I was gonna propose something a bit different but given the complexity I think this approach makes more sense indeed!

Can you please just add a comment above this line saying `// Google requires to not pass the price when the price status is NOT_CURRENTLY_KNOWN`?

Reply:
Added.

Reply:
I just updated the branch and will merge the fix, thanks a lot for your contribution!